src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommand.cs

```diff
@@ -278,8 +278,8 @@ private void RunScriptSilentlyAndWithErrorHookup(string script)
             // errors are not created here, because there is a field for it used in the final pop up
             PSDataCollection<object> output = new PSDataCollection<object>();
 
-            output.DataAdded += new EventHandler<DataAddedEventArgs>(this.Output_DataAdded);
-            _errors.DataAdded += new EventHandler<DataAddedEventArgs>(this.Error_DataAdded);
+            output.DataAdded += this.Output_DataAdded;
```

Comment:
This still seems like a different type of fix to me.

Reply:
These changes are now presented in #13441.